src/components/DocumentationTopic/DocumentationNav/Hierarchy.vue

```diff
@@ -54,13 +62,14 @@
 import { buildUrl } from 'docc-render/utils/url-helper';
 import NavMenuItems from 'docc-render/components/NavMenuItems.vue';
 import Badge from 'docc-render/components/Badge.vue';
+import throttle from 'docc-render/utils/throttle';
 import HierarchyCollapsedItems from './HierarchyCollapsedItems.vue';
 import HierarchyItem from './HierarchyItem.vue';
 
-// The max number of items that will initially be visible and uncollapsed w/o
+// The max number of link items that will initially be visible and uncollapsed w/o
 // any user interaction. If there are more items, they will be collapsed into a
 // menu that users need to interact with to see.
-const MaxVisibleItems = 3;
+const MaxVisibleLinks = 3;
```

Comment:
Can you explain this more? Is it 3 parents? I see more than 3 links in the NavBar here: http://localhost:8080/documentation/slothcreator/sloth/color-swift.enum/hash(into:)

Reply:
I will add a bit more explanation to the code. We essentially have `root / technology / [...] / link_1 / link_2 / link_3 / current_page`, and based on resolution, we start pushing `link_1,2,3` into the `[...]` collapsable items.